src/askui/tools/askui/askui_controller.py

```diff
@@ -1,40 +1,108 @@
 from typing import List, Literal
 import grpc
-import askui.tools.askui.askui_ui_controller_grpc.Controller_V1_pb2_grpc as controller_v1
-import askui.tools.askui.askui_ui_controller_grpc.Controller_V1_pb2 as controller_v1_pbs
+import os
 
 import time
 from PIL import Image
 
 import subprocess
 import uuid
-import os
 import time
 import sys
+import json
 import time
 import subprocess
 
 from ..utils import process_exists, wait_for_port
 from askui.reporting.report import SimpleReportGenerator
 from askui.utils import draw_point_on_image
 
+import askui.tools.askui.askui_ui_controller_grpc.Controller_V1_pb2_grpc as controller_v1
+import askui.tools.askui.askui_ui_controller_grpc.Controller_V1_pb2 as controller_v1_pbs
+
+def load_json_file(file_path: str) -> dict:
+    """
+    Loads a JSON file and returns its contents as a dictionary.
+    
+    Args:
+        file_path (str): Path to the JSON file
+        
+    Returns:
+        dict: Contents of the JSON file
+        
+    Raises:
+        FileNotFoundError: If the file doesn't exist
+        json.JSONDecodeError: If the file contains invalid JSON
+    """
+    try:
+        with open(file_path, 'r') as file:
+            return json.load(file)
+    except FileNotFoundError:
+        print(f"Error: File '{file_path}' not found")
+        raise
+    except json.JSONDecodeError as e:
+        print(f"Error: Invalid JSON in file '{file_path}': {str(e)}")
+        raise
```

Comment:
Would remove the print statements as we let the error bubble up anyway or raise new errors with all info included, e.g., original error + file path, or at least replace the prints using proper logging

Reply:
Gone.